consensus/spos/bls/v2/subroundEndRound.go

```diff
@@ -501,24 +509,23 @@ func (sr *subroundEndRound) handleInvalidSignersOnAggSigFail() ([]byte, []byte,
 	invalidPubKeys, err := sr.verifyNodesOnAggSigFail(ctx)
 	cancel()
```

Comment:
not related to the changes from this PR, but this cancel() doesn't seem right to call it after the "intensive". Maybe move it up 1 line and call it on a defer?